dvc/command/get.py

```diff
@@ -40,10 +42,12 @@ def add_parser(subparsers, parent_parser):
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
     get_parser.add_argument(
-        "url", help="URL of Git repository with DVC project to download from."
+        "url",
+        help="Location of DVC project or Git repository to download from.",
```

Comment:
Not a complete sentence so no period
```suggestion
        help="Location of DVC project or Git repository to download from",
```